### PR TITLE
fix: cannot select first item in global search

### PR DESCRIPTION
**Pull request by lukicenturi**
Closes #(issue_number)

## Checklist

- [ ] The PR modified the frontend, and updated the [user guide](https://github.com/rotki/rotki/blob/develop/docs/usage_guide.rst) to reflect the changes.

### CODE REVIEW

**Review by cleanerzkp — APPROVED** [review]
LGTM!

File: frontend/app/src/components/GlobalSearch.vue [hunk 1]
```diff
@@ -376,7 +376,7 @@ watch(open, (open) => {
 });
 
 function change(index?: number) {
-  if (!index)
+  if (!isDefined(index))
```
[the remark this answers is not in the document]
Comment (kelsos) [reply frontend/app/src/components/GlobalSearch.vue:379]: Or it's just as an alternative to `if(index !== undefined)`

**Review by kelsos — APPROVED** [review]
lgtm

File: frontend/app/src/components/GlobalSearch.vue (hunk 1 above)
[the remark this answers is not in the document]
Comment (lukicenturi) [reply frontend/app/src/components/GlobalSearch.vue:379]: yes `if(index !== undefined)` this actually

**Pull request merged**